Godnoken/Preprocessing-II
# Preprocessing 2

## What does this PR do?
Preprocessing 2 I MVP
​
## Description of tasks completed
- Implemented Task 3: Import LESS Files into index.less
- Implemented Task 4: Update files to complete desktop layout
- Implemented Task 5: Update files to add mobile layout (500px)
​
## How can this be tested?
​1. Clone the project.
2. Check index.less for @import
3. Open up index.html and check desktop layout at any size larger than 500px width.
4. Resize browser window to 500px to check mobile layout.

## Relevant screenshot
![image](https://user-images.githubusercontent.com/17499594/66579209-bf54f980-eb7c-11e9-9885-56f0678196f4.png)

![image](https://user-images.githubusercontent.com/17499594/66579269-d98ed780-eb7c-11e9-911f-6db625b73eb0.png)

![image](https://user-images.githubusercontent.com/17499594/66579296-e90e2080-eb7c-11e9-8431-98a04ed78766.png)
